modules/protect/math-fallback.php

```diff
@@ -105,22 +106,37 @@ public function process_generate_math_page() {
 		/**
 		 * Requires a user to solve a simple equation. Added to any WordPress login form.
 		 *
-		 * @return VOID outputs html
+		 * @param boolean $echo Default to echo and not return the equation fields.
+		 *
+		 * @return VOID|String outputs or returns html
 		 */
-		static function math_form() {
+		static function math_form( $echo = true ) {
 			$salt = get_site_option( 'jetpack_protect_key' ) . get_site_option( 'admin_email' );
 			$num1 = rand( 0, 10 );
 			$num2 = rand( 1, 10 );
 			$sum  = $num1 + $num2;
 			$ans  = sha1( $salt . $sum );
-			?>
-			<div style="margin: 5px 0 20px;">
-				<strong><?php esc_html_e( 'Prove your humanity:', 'jetpack' ); ?> </strong>
-				<?php echo $num1 ?> &nbsp; + &nbsp; <?php echo $num2 ?> &nbsp; = &nbsp;
-				<input type="input" name="jetpack_protect_num" value="" size="2" />
-				<input type="hidden" name="jetpack_protect_answer" value="<?php echo $ans; ?>" />
-			</div>
-		<?php
+
+			$output = '<strong>' . esc_html( 'Prove your humanity:', 'jetpack' ) . ' </strong>' .
```

Comment:
@hereswhatidid this `esc_html` should be `esc_html__` otherwise the string is not available for translation